gdx/src/com/badlogic/gdx/graphics/g2d/PixmapPacker.java

```diff
@@ -101,6 +101,7 @@
 	Format pageFormat;
 	int padding;
 	boolean duplicateBorder;
+	Color currentTransparentColor = new Color(0f, 0f, 0f, 0f);

```

Comment:
Why not just transparentColor/getTransparentColor/setTransparentColor? I think that one can reasonably assume that they are getting/setting the current value.